gaurun/apns_http2.go

```diff
@@ -66,7 +66,7 @@ func loadX509KeyPairWithPassword(certPath, keyPath, keyPassphrase string) (tls.C
 	if keyPassphrase != "" {
 		pemBlock, _ := pem.Decode(keyPEMBlock)
 		if !x509.IsEncryptedPEMBlock(pemBlock) {
-			err = errors.New(keyPath + "is not encrypted. passphrase is not required")
+			err = fmt.Errorf("%s(private-key-pem-file-path) is not encrypted. passphrase is not required",keyPath)
```

Comment:
Sorry. My word was not good. The string of `(private-key-pem-file-path)` is not required.

And please run `go fmt` before commiting changes. Your latest code is not go-formated.

```
$ make fmt
gaurun/apns_http2.go
$
```